src/channel/pusher-presence-channel.ts

```diff
@@ -5,16 +5,15 @@ import { PresenceChannel } from './presence-channel';
  * This class represents a Pusher presence channel.
  */
 export class PusherPresenceChannel extends PusherChannel implements PresenceChannel {
-
     /**
      * Register a callback to be called anytime the member list changes.
      *
      * @param  {Function} callback
      * @return {object} this
      */
     here(callback): PusherPresenceChannel {
-        this.bind('pusher:subscription_succeeded', (data) => {
-            callback(Object.keys(data.members).map(k => data.members[k]), this.channel);
+        this.on('pusher:subscription_succeeded', (data) => {
+            callback(Object.keys(data.members).map(k => data.members[k]), this.name);
```

Comment:
Ok, I will remove this. Each channel now has a "subscription" which is the root object of the socket from which ever provider. (pusher or socket.io)